templates/nextflow-ecs-task-definition.yaml

```diff
@@ -102,15 +102,24 @@ Parameters:
     Type: Number
     Description: (Optional) Port to open on host for backend container
     Default: 8080
+  EfsFileSystemId:
+    Type: String
+    Description: The ID of the EFS file system to mount as a volume
+  EfsVolumeMountPath:
+    Type: String
+    Description: Path where the EFS volume container tower configuration is mounted
+  TowerConfigFileName:
+    Type: String
+    Description: Name of the tower config YAML file
 Resources:
   TowerTask:
     Type: AWS::ECS::TaskDefinition
     Properties:
       NetworkMode: bridge
       Volumes:
-        - Name: tower_yaml
-          Host:
-            SourcePath: '/opt/nextflow/tower.yml'
+        - Name: efs_volume
```

Comment:
`efs_volume` string appears a few times in this file.  how about making it a parameter?

Reply:
parameterized in c4506b4